recipes/boost/all/conanfile.py

```diff
@@ -801,7 +801,15 @@ def _bootstrap(self):
                         option = "" if tools.os_info.is_windows else "-with-toolset="
                         cmd = "%s %s%s" % (bootstrap, option, self._get_boostrap_toolset())
                     self.output.info(cmd)
-                    with tools.environment_append({"CC": None, "CXX": None, "CFLAGS": None, "CXXFLAGS": None}):
+
+                    removed_environment_variables = dict(((_, None) for _ in ("CC",
```

Comment:
Use a different name for `_`. Something like `var` or similar. Normally `_` name is used in python to indicate that the variable is not needed or not going to be used